internal/testutil/pkgdata.go

```diff
@@ -13,150 +12,31 @@ import (
 
 var PackageData = map[string][]byte{}
 
-func init() {
-	baseFilesData, err := base64.StdEncoding.DecodeString(baseFilesBase64)
-	if err != nil {
-		panic("broken package base64")
-	}
-	PackageData["base-files"] = baseFilesData
+var testPackageEntries = []TarEntry{
+	Dir(0755, "./"),
+	Dir(0755, "./dir/"),
+	Reg(0644, "./dir/file", "12u3q0wej	ajsd"),
+	Reg(0644, "./dir/other_file", "kasjdf0"),
+	Dir(0755, "./dir/nested/"),
+	Reg(0644, "./dir/nested/file", "0jqei"),
+	Reg(0644, "./dir/nested/other_file", "1"),
+	Dir(0755, "./dir/several/levels/deep/"),
+	Reg(0644, "./dir/several/levels/deep/file", "129i381		"),
+	Dir(0755, "./other_dir/"),
```

Comment:
The overall naming strategy looks nice, thanks.  There's one detail which would be nice to preserve, though, since this will likely create inconsistency here and in other places: the convention used for naming files and directories in the project is typically with dashes, not underlines.  Underlines are typically used as a stronger separation between components which are themselves dashes. E.g. test-package_slice.

Can we please tune the file list after that?  Sorry for the additional trouble.

Reply:
Done, I have changed all occurrences in the package and in the tests themselves.